apache/couchdb-fauxton
# Fix Replication urls

## Overview

Fixed the url for replication

## Testing recommendations

1. Login in Fauxton
2. Click on a database
3. At the top left side bar, click on the three dots to the right of the database name (ViewTabsSideBar).
4. Click on `Replicate Database`
5. It should route to the replication view.

## GitHub issue number

#1007 

## Related Pull Requests



## Checklist
